### PR TITLE
2432: Improve performance when selecting asset

### LINKED ISSUE #2384: Increase default number of rows for assets page
For the Planning Assets table, use a default of 100 assets.

Check table performance with 250, 500, 1000 items.

@lebaphi has established that the performance really slows down in the assets table with lots of rows.

I've been playing and have (slightly) reduce the number of events that get triggered from the table.

I have a suspicion the table is doing lots and lots of rendering.  We have the `render-icon` and `render-platform` - which both do a bit of processing.

### PR DESCRIPTION
**Pull request by lebaphi**
Fixes #2384 #2386 #2432 

Hi @IanMayo, I just made some changes to reduce rendering, it seems the performance has improved (it only takes 1~2 seconds when this table loads 250 rows)

https://user-images.githubusercontent.com/26705621/219589492-360f1a60-bfd5-4e76-aea5-6ca28c2a37bf.mov

But I found that the material table itself has an performance issue when enable/disable the filter, it makes 3~5s to applied, I'm investigating it.

### CODE REVIEW

**Comment by IanMayo** [comment]
Brilliant!

It looks like it needs a bit more review - will do that later.

**Comment by IanMayo** [comment]
@lebaphi - you've lost some of my changes 🤣 

For the material tables, I have added a new tool button for `planning-messages`, and provided two-state icons for `planning-messages` and `planning-assets`.  The original bits are in red in `Files Changed`.

File: packages/components/src/local/organisms/planning-channel/index.tsx [hunk 1]
```diff
@@ -366,23 +370,16 @@ export const PlanningChannel: React.FC<PropTypes> = ({
...
-    // only update selected assets if we're not planning an activity
-    if (!activityBeingPlanned) {
-      setSelectedAssets(localSelectedAssets)
-    }
-  }, [localSelectedAssets])
-
   useEffect(() => {
     // produce the own and opp assets for this player force
     const forceCols = getForceColors(allForces)
     const platIcons = platformIcons(platformTypes)
```
Comment (IanMayo) [inline packages/components/src/local/organisms/planning-channel/index.tsx:376]: hello @lebaphi -  your code has dropped the extra parameter for the next two calls - that will prevent some functionality from working.

**Comment by lebaphi** [comment]
Close due to this PR https://github.com/serge-web/serge/pull/2444

**Pull request closed without merging**